cognite/client/_api/workflows.py

```diff
@@ -47,6 +54,130 @@ def wrap_workflow_ids(
     return WorkflowIds.load(workflow_version_ids).dump(camel_case=True, as_external_id=True)
 
 
+class BetaWorkflowAPIClient(APIClient, ABC):
+    def __init__(
+        self,
+        config: ClientConfig,
+        api_version: str | None,
+        cognite_client: CogniteClient,
+    ) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._api_subversion = "beta"
+        self._warning = FeaturePreviewWarning(
+            api_maturity="beta", sdk_maturity="beta", feature_name="Workflow Orchestration"
+        )
+
+
+class WorkflowTriggerAPI(BetaWorkflowAPIClient):
+    _RESOURCE_PATH = "/workflows/triggers"
```

Comment:
Why this parent class here? Why just add it to the `WorkflowTriggerAPI`? Then you do not have to make any exception in the mock test.

```suggestion
class WorkflowTriggerAPI(APIClient):
    _RESOURCE_PATH = "/workflows/triggers"

    def __init__(
        self,
        config: ClientConfig,
        api_version: str | None,
        cognite_client: CogniteClient,
    ) -> None:
        super().__init__(config, api_version, cognite_client)
        self._api_subversion = "beta"
        self._warning = FeaturePreviewWarning(
            api_maturity="beta", sdk_maturity="beta", feature_name="Workflow Orchestration"
        )
```